### PR TITLE
Anastasiia pedorenko hw 4

### PR DESCRIPTION
**Pull request by ananas111**

### CODE REVIEW

File: Pedorenko/HW4/task1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+year = int(input("input year"))
+if (year % 4 == 0 and year % 100 != 0) or year % 400 == 0:
+    print('this year is intercalary')
+else:
+    print("this year is common")
+century = year // 100 + 1
+print(f"year belongs to {century} century")
```
Comment (katemalash) [inline Pedorenko/HW4/task1.py:7]: Good Decision

File: Pedorenko/HW4/task2.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+geometric_shapes = str(
```
Comment (katemalash) [inline Pedorenko/HW4/task2.py:1]: is it mandatory here to cast to `str()`?

File: Pedorenko/HW4/task2.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    input("input geometric shape from choice: rectangle, triangle or circle "))
+if geometric_shapes == 'rectangle' or geometric_shapes == 'triangle':
```
Comment (katemalash) [inline Pedorenko/HW4/task2.py:3]: The formula for triangle square is not the same as the formula for rectangle square (Heron's formula). Please fix these calculations

File: Pedorenko/HW4/task3.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+number = int(input('input number '))
+if number == 0:
+    print("null")
+elif number < 0:
+    number_of_digits = len(str(number)) - 1
+    status = 'negative'
+else:
+    number_of_digits = len(str(number))
+    status = 'positive'
+print(f"{status}{number_of_digits}-digits")
```
Comment (katemalash) [inline Pedorenko/HW4/task3.py:10]: Good Decision

File: Pedorenko/HW4/task4.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+radius = int(input("enter radius R"))
+square_hall = int(input("enter square S"))
+pi = 3.14159265359
+square_stage = pi * radius ** 2
+K = int(input("enter K"))
+if square_stage < square_hall and ((square_hall ** 0.5 / 2) - radius) >= K:
```
Comment (katemalash) [inline Pedorenko/HW4/task4.py:6]: Is the first condition mandatory here?

File: Pedorenko/HW4/task5.py [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+        banknote100 = money // 100
+        money -= banknote100 * 100
+        if money // 10 >= 0:
+            banknote10 = money // 10
+            money -= banknote10 * 10
+            if money // 1 >= 0:
+                banknote1 = money // 1
+
+print(
+    f"we can exchange money into {banknote200} banknotes by 200 UAH, {banknote100} banknotes by 100 UAH, {banknote10} banknotes by 10 UAH, {banknote1} banknotes by 1 UAH ")
```
Comment (katemalash) [inline Pedorenko/HW4/task5.py:15]: Good Decision

File: Pedorenko/HW4/task6.py [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+    if place % 2 == 0:
+        print('you have a top place')
+    else:
+        print('you have a bottom place')
+    if place < 37:
+        print('your place is compartment')
+    else:
+        print('your place is side')
+else:
+    print('you entered the wrong number')
```
Comment (katemalash) [inline Pedorenko/HW4/task6.py:12]: Good Solution

**Review by katemalash — CHANGES_REQUESTED** [review]
Please fix the calculations for the triangle's square

File: Pedorenko/HW4/task2.py (hunk 3 above)
Comment (ananas111) [reply Pedorenko/HW4/task2.py:3]: I fixed it, thanks!

File: Pedorenko/HW4/task2.py (hunk 2 above)
Comment (ananas111) [reply Pedorenko/HW4/task2.py:1]: nope

File: Pedorenko/HW4/task4.py (hunk 5 above)
Comment (ananas111) [reply Pedorenko/HW4/task4.py:6]: nope

**Comment by ananas111** [comment]
I edited the homework and committed the result.

**Review by katemalash — APPROVED** [review]
Good Job. Please merge this pull request

**Pull request merged**